openapi.yaml

```diff
@@ -447,7 +447,7 @@ components:
         kessel.inventory.v1beta1.relationships.DeleteK8SPolicyIsPropagatedToK8SClusterRequest:
             type: object
             properties:
-                reporter_data:
+                reporterData:
```

Comment:
all of these are going to need an updated in their proto files e.g. `json_name=reporter_data`

Reply:
done and the PR has been updated